app/tests/forecasts/test_forecasts_db.feature

```diff
@@ -0,0 +1,11 @@
+Feature: /forecasts/
```

Comment:
I know you're gonna hate me for this because it's a lot of tedious refactoring, but Kyubin and I were discussing earlier this week how the /forecasts/ route is a misleading name, because it relates to models, not meteorologist's forecasts. Kyubin already did some work as part of my PR to replace "forecasts" with "models". Could we replace these "forecasts" with "models" too?

Reply:
Not a problem at all - I've been thinking the same.

Reply:
Actually - /models/{model}/forecasts/ - so you can do /models/gdps/forecasts/

Reply:
yeah I like that even better